### DISCUSSION TITLE
How can I get the secret trophie ?

### DISCUSSION THREAD

**Discussion by parmeshwar1820** [anuraghazra/github-readme-stats#2207] [Q&A] [answered]
How can I get the secret trophie ?

**Comment by DURGESH716** [comment, marked as answer]
In order to get the secret trophy, you have to come up with some interesting conditions, like first I got the "Multi Lang User" trophy, but later it got removed.

**Reply by rickstaa** [reply]
I didn't know we had secret trophies. 😅 I think this question was meant to be asked at https://github.com/ryo-ma/github-profile-trophy.

**Reply by parmeshwar1820** [reply]
Yeah we have !!

**Reply by rickstaa** [reply]
@parmeshwar1820 It is nowehere in the codebase. You're sure you're not talking about the https://github.com/ryo-ma/github-profile-trophy repository.
